### PR TITLE
Feature: Hint cover drag for first time users

### PR DESCRIPTION
**Pull request by pichfl**
This enables a drag hint that nudges the artwork every time the player is presented until the user uses the drag gesture to rewind or forward.

The same nudge is used when the icons behind the artwork are tapped in an attempt to jump that way.

This should provide enough feedback and encouragement to users that return from an earlier version and also convey a little achievement when they interact with the artwork the first time.

Re #196

### CODE REVIEW

**Comment by pichfl** [comment]
I had to get a bit create to match the play/pause button to the artwork size in paused mode as simply scaling the button would have changed the scale of the play button which I didn't intend to. 

I'm setting a new key to user defaults and since I suspect user defaults to be a bit on the slow side, I only read and set them only indirectly. That may not be necessary, but I tried to match what we have elsewhere in the app.

File: BookPlayer/Player/Containers/PlayerControlsViewController.swift [hunk 1]
```diff
@@ -86,8 +102,12 @@ class PlayerControlsViewController: PlayerContainerViewController, UIGestureReco
         NotificationCenter.default.addObserver(self, selector: #selector(self.onPlayback), name: Notification.Name.AudiobookPlayer.bookPlaying, object: nil)
     }
 
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        if !UserDefaults.standard.bool(forKey: UserDefaultsConstants.artworkJumpControlsUsed) {
```
Comment (GianniCarlo) [inline BookPlayer/Player/Containers/PlayerControlsViewController.swift]: why not use `self.artworkJumpControlsUsed`? it's already intialized in `viewDidLoad`

File: BookPlayer/Player/Views/ArtworkControl.swift [hunk 2]
```diff
@@ -145,17 +151,25 @@ class ArtworkControl: UIView, UIGestureRecognizerDelegate {
...
         self.pan.cancelsTouchesInView = true
 
         self.addGestureRecognizer(self.pan!)
-    }
 
-    override func layoutIfNeeded() {
+        self.tap = UITapGestureRecognizer(target: self, action: #selector(tapAction))
+        self.tap.delegate = self
+        self.tap.numberOfTapsRequired = 1
+        self.tap.numberOfTouchesRequired = 1
```
Comment (GianniCarlo) [inline BookPlayer/Player/Views/ArtworkControl.swift]: default value is already `1` https://developer.apple.com/documentation/uikit/uitapgesturerecognizer/1623580-numberoftouchesrequired

File: BookPlayer/Player/Containers/PlayerControlsViewController.swift (hunk 1 above)
Comment (pichfl) [reply BookPlayer/Player/Containers/PlayerControlsViewController.swift]: Doh! Forgot to refactor that one

**Pull request merged**